engines/scumm/bomp.cpp

```diff
@@ -64,12 +64,17 @@ void bompDecodeLine(byte *dst, const byte *src, int len) {
 		len -= num;
 		if (code & 1) {
 			color = *src++;
-			memset(dst, color, num);
+			if (setZero || color)
+				memset(dst, color, num);
+			dst += num;
 		} else {
-			memcpy(dst, src, num);
```

Comment:
It might be better to continue using `memcpy` when `setZero` is true.

Reply:
thank you, done